### PR TITLE
Add Mail for Good

### PR DESCRIPTION
**Pull request by karlcoelho**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language` ``

- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 1 entries with syntax error(s).</td>
    </tr>
  </tbody>
</table>

#### Syntax Issues

 Name | Entry
----|----------------------
Mail For Good |   * [Mail For Good](https://github.com/freeCodeCamp/mail-for-good) - An open source email campaign management tool for nonprofits  `BSD` `Javascript` 
 

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by karlcoelho** [comment]
I can't tell what is the syntax error here. Can someone help me?

**Comment by karlcoelho** [comment]
Still cannot tell the error.

**Comment by nodiscc** [comment]
Mail For Good either requires deploying on Amazon's AWS, or relies on Amazon SES for "local" installation (https://github.com/freeCodeCamp/mail-for-good/blob/master/docs/local_deploy.md). I don't think this qualifies as self-hosted. @Kickball do you agree?

**Comment by karlcoelho** [comment]
Just added the  `⚠` tag to Mail for Good addition. All good now?

**Comment by karlcoelho** [comment]
@nodiscc Think you can help me here? I'm not sure what I'm doing wrong.

File: README.md [hunk 1]
```diff
@@ -304,6 +304,7 @@ _MDAs - IMAP/POP3 software_
 _Mailing lists servers and mass mailing software - one message to many recipients._
 
   * [Dada Mail](http://dadamailproject.com/) - A web-based list management system that can be used for announcement lists and/or discussion lists. ([Source Code](https://github.com/justingit/dada-mail)) `GPLv2` `Perl`
+  * [Mail For Good](https://github.com/freeCodeCamp/mail-for-good) `⚠`  - An open source email campaign management tool for nonprofits.  `BSD` `Javascript`
```
Comment (nodiscc) [inline README.md]: > Just added the ⚠ tag to Mail for Good addition. All good now?

All right, let's do that.

Syntax errors:

 * double space after `⚠`
 * double space before license

**Comment by karlcoelho** [comment]
@nodiscc Fix the spacing issue. Can we move forward with this addition?

**Comment by karlcoelho** [comment]
Are we going to proceed with this PR? @nodiscc @Kickball

**Pull request merged**